neural_style.py

```diff
@@ -10,15 +10,25 @@
 import math
 from argparse import ArgumentParser
 
+from matplotlib import pyplot as plt
+
+import tensorflow as tf
```

Comment:
Fixed in https://github.com/anishathalye/neural-style/pull/50/commits/cdf30af1b40bdd13c79494eaad106186fe25e762